common/utils/utils.go

```diff
@@ -170,6 +170,43 @@ func PreloadGCSFiles(tables []ManifestTable) ([]ManifestTable, error) {
 	return tables, nil
 }
 
+func WriteToGCS(filePath, fileName, data string) error {
+	ctx := context.Background()
+
+	client, err := storage.NewClient(ctx)
+	if err != nil {
+		fmt.Printf("Failed to create GCS client")
+		return err
+	}
+	defer client.Close()
+	if filePath[len(filePath)-1] != '/' {
```

Comment:
This could break if the length of filePath is 0 , would be better to handle that case as well?

Reply:
Done